### DISCUSSION TITLE
Variable via config for wait time before speech input is complete

### DISCUSSION THREAD

**Discussion by jaswalters** [toverainc/willow#171] [Ideas]
I've noticed on occasion when I say Hi ESP and its waking up from a sleep I will say an entire sentence but I guess it thinks I'm done even though I provided no pause in the verbal communication. I'm wondering if there needs to be a variable we can set on how long it waits for no input before processing the input?

**Comment by stintel** [comment]
Already there: https://github.com/toverainc/willow/blob/feature/multiple-wake/main/Kconfig.projbuild#L113-L121
